packages-native/intro-screen/src/IntroScreen.xml

```diff
@@ -0,0 +1,131 @@
+                                <description/>
+                            </property>
+                        </propertyGroup>
+                    </properties>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Display">
+                <property key="showMode" type="enumeration" defaultValue="fullscreen">
+                    <caption>Show as</caption>
+                    <description />
+                    <enumerationValues>
+                        <enumerationValue key="fullscreen">Fullscreen</enumerationValue>
+                        <enumerationValue key="popup">Popup</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="buttonPattern" type="enumeration" defaultValue="all">
+                    <caption>Buttons</caption>
+                    <description>The caption and icon of the buttons change dynamically based on which slide the user is. For example on the first page a skip button is shown but now a previous button.</description>
+                    <enumerationValues>
+                        <enumerationValue key="all">Skip / Previous and Next / Done</enumerationValue>
+                        <enumerationValue key="next_done">Only Next / Done</enumerationValue>
+                        <enumerationValue key="none">None</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="slideIndicators" type="enumeration" defaultValue="between">
+                    <caption>Show indicator</caption>
+                    <description/>
+                    <enumerationValues>
+                        <enumerationValue key="between">Between buttons</enumerationValue>
+                        <enumerationValue key="above">Above buttons</enumerationValue>
+                        <enumerationValue key="never">Never</enumerationValue>
+                    </enumerationValues>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Common">
+                <systemProperty key="Visibility" />
+                <systemProperty key="Name"/>
+            </propertyGroup>
+        </propertyGroup>
+        <propertyGroup caption="Buttons">
+
+            <propertyGroup caption="Skip button">
+                <property key="skipCaption" type="textTemplate" required="false">
+                    <caption>Caption</caption>
+                    <description/>
+                </property>
+                <property key="skipIcon" type="icon" required="false">
+                    <caption>Skip Icon</caption>
+                    <description/>
+                </property>
+            </propertyGroup>
+            <propertyGroup caption="Previous button">
+                <property key="previousCaption" type="textTemplate" required="false">
+                    <caption>Caption</caption>
+                    <description/>
```

Comment:
should be use language default value here? Check with Danny as requirement
``` xml
<translations>
  <translation lang="en_US">Previous</translation>
  <translation lang="nl_NL">Vorige</translation>
</translations>
```
https://docs.mendix.com/apidocs-mxsdk/apidocs/property-types-pluggable-widgets#4-4-3-studio-pro-ui